src/main/scala/edu/washington/cs/seguard/js/JSFlowGraph.scala

```diff
@@ -192,7 +202,19 @@ object JSFlowGraph {
           }
 
           // DefUse based analysis
-          val u = currentNode.get
+          var u = currentNode.get
+          val namespace = bb.getNode.toString
+          if (!globalVarMap.contains(namespace)) {
+            globalVarMap.put(namespace, new HashMap());
```

Comment:
what is `namespace`?

Reply:
Here namespace refers to the name of the function where these variables are defined. For example, in different call graph node (or, in other words, function), v11 may refer to different variables.

Reply:
Okay, I see. Please add this comment to code.